### PR TITLE
feat: add auto-completion to status command to retrieve clusters names

### LINKED ISSUE #3874: [Feature]: Auto completion for the cnpg plugin
### Is there an existing issue already for this feature request/idea?

- [X] I have searched for an existing issue, and could not find anything. I believe this is a new feature request to be evaluated.

### What problem is this feature going to solve? Why should it be added?

Usually, Kubernetes users use to type <TAB> twice to get a help or a completion of whats required for a command, for example, that works like this:
```
kubectl get pods <TAB><TAB>
```
This will retrieve a list of pods, which it's absent in our case

### Describe the solution you'd like

When typing the following command:
```
kubectl cnpg status <TAB><TAB>
```
I would like to have a completion list of the clusters in the selected namespace and not a list of the file sin my current directory

### Describe alternatives you've considered

None

### Additional context

_No response_

### Backport?

Yes

### Are you willing to actively contribute to this feature?

No

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by sxd**
Users are used to do `kubectl get <TAB><TAB>` to get help, this capability was missing in the CNPG plugin, with this new and initial feature, we will have that capability and the starting point to add it in other sub-commands.

Closes #3874

### CODE REVIEW

File: docs/src/kubectl-plugin.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ For example, let's install the 1.18.1 release of the plugin, for an Intel based
 64 bit server. First, we download the right `.deb` file.
 
 ``` sh
-$ wget https://github.com/cloudnative-pg/cloudnative-pg/releases/download/v1.18.1/kubectl-cnpg_1.18.1_linux_x86_64.deb
+wget https://github.com/cloudnative-pg/cloudnative-pg/releases/download/v1.18.1/kubectl-cnpg_1.18.1_linux_x86_64.deb
 ```
```
Comment (gbartolini) [inline docs/src/kubectl-plugin.md:38]: Why 1.18?

Comment (armru) [reply docs/src/kubectl-plugin.md:38]: Should this match with the build version?

Comment (mnencia) [reply docs/src/kubectl-plugin.md:38]: I believe this is just a missing place where your release script should have upgraded the version.

Comment (sxd) [reply docs/src/kubectl-plugin.md:38]: yeah for sure, because I only removed the $ because testing the copy&paste failed

**Review by leonardoce — APPROVED** [review]

**Comment by leonardoce** [comment]
E2e: https://github.com/EnterpriseDB/cloudnative-pg/actions/runs/8250811343

**Pull request merged**